.teamcity/_self/projects/WebApp.kt

```diff
@@ -214,6 +214,18 @@ object RunCalypsoE2eMobileTests : BuildType({
 			"""
 			dockerImage = "%docker_image_e2e%"
 		}
+		bashNodeScript {
+			name = "Update Chrome"
+			scriptContent = """
+				wget --no-verbose https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_90.0.4430.93-1_amd64.deb
```

Comment:
This needs to be in the same step than `Run e2e tests (mobile)`. Otherwise it won't get persisted.

IIRC, TeamCity creates a new container on every step (🤦🏼 ), and the only directory persisted between them is the checkout directory.

Reply:
I was wondering about this I tried not to look to hard up the stack, will move it over :bow: